src/ip_info.rs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2022 MobileCoin Foundation
+
+use crate::{Error, Location, LocationProvider};
+use reqwest::{
+  blocking::Client,
+  header::{HeaderMap, HeaderValue, CONTENT_TYPE},
+};
+
+pub struct IpInfoIoFetch;
+
+impl LocationProvider for IpInfoIoFetch {
+  fn location(&self) -> Result<Location, Error> {
+    let mut json_headers = HeaderMap::new();
+    json_headers.insert(CONTENT_TYPE, HeaderValue::from_static("application/json"));
```

Comment:
We should support passing an auth token, like https://github.com/mobilecoinfoundation/mobilecoin/pull/1996. Could be in a follow-up PR.

Reply:
Yeah, I've opened #3 to handle this (and similar) concerns.